### PR TITLE
Added Slovak Language Pack

### LINKED ISSUE #2667: Import Slovak from old repo
https://github.com/AnySoftKeyboard/LanguagePack/tree/Slovak

### PR DESCRIPTION
**Pull request by Aiq0**
Added Slovak Language Pack (mainly imported from https://github.com/AnySoftKeyboard/LanguagePack/tree/Slovak)

Fixed #2667

### CODE REVIEW

File: addons/languages/slovak/apk/build.gradle [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```
Comment (menny) [inline addons/languages/slovak/apk/build.gradle:1]: I assume this pack is the published pack at https://play.google.com/store/apps/details?id=org.herrlado.ask.languagepack.slovak

If so, you need to also override the application ID to match `org.herrlado.ask.languagepack.slovak`

**Review by menny — CHANGES_REQUESTED** [review]
Thank you for migrating! 

I had one comment

File: addons/languages/slovak/apk/build.gradle (hunk 1 above)
Comment (Aiq0) [reply addons/languages/slovak/apk/build.gradle:1]: Is it OK now?

Comment (Aiq0) [reply addons/languages/slovak/apk/build.gradle:1]: @menny, is it OK now?

Comment (menny) [reply addons/languages/slovak/apk/build.gradle:1]: Yes. Thanks

**Comment by menny** [comment]
https://github.com/AnySoftKeyboard/AnySoftKeyboard/runs/3790941669#step:6:1394 there are some style issues.

**Comment by Aiq0** [comment]
@menny it should be fixed now with the latest commit

**Review by menny — APPROVED** [review]

**Pull request merged**